### PR TITLE
Fossa issues

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:
Fixes [FOSSA](fossa.com) licensing issues brought up [here](https://app.fossa.com/projects/git%2Bgithub.com%2Fkubevirt%2Fkubevirt/refs/branch/master/9d222d569dbedb7aa06a68e2dd6312f29c115fc0)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Fix FOSSA licensing issues
```

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -414,6 +424,8 @@ k8s.io/kubernetes v1.11.7-beta.0.0.20181219023948-b875d52ea96d/go.mod h1:ocZa8+6
 k8s.io/kubernetes v1.11.8-beta.0.0.20190124204751-3a10094374f2/go.mod h1:ocZa8+6APFNC2tX1DZASIbocyYT5jHzqFVsY5aoB7Jk=
 k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a h1:2jUDc9gJja832Ftp+QbDV0tVhQHMISFn01els+2ZAcw=
 k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a/go.mod h1:sZAwmy6armz5eXlNoLmJcl4F1QuKu7sr+mFQ0byX7Ew=
+kubevirt.io/containerized-data-importer v1.8.1-0.20190516083534-83c12eaae2ed/go.mod h1:qF594BtRRkruyrqLwt3zbLCWdPIQNs1qWh4LR1cOzy0=
```
Comment (slintes) [inline go.sum]: not sure about all the other changes, but this one should not be here, we updated cdi to 1.10.6 (see line below). What did you do for modifying these files? `make deps-update` should be the way to go.

Comment (dhiller) [reply go.sum]: I need to rebase on latest master anyway. I'll revert the changes and use `make deps-update` and see how it goes.

Comment (dhiller) [reply go.sum]: `make deps-update` didn't update any dependency locally, I see internally it does 

```
go mod tidy 
go mod vendor
```

which doesn't update deps, does it? Therefore I used `go get -u ...`. Please elaborate?

Comment (dhiller) [reply go.sum]: To elaborate on this: this is about updating dependencies to mitigate licensing issues. I.e. for the/assert we are using an old version without license, where the new one does have one.

Comment (slintes) [reply go.sum]: Ack. `go get -u ... ` is correct for updating versions. But still please run `make deps-update` afterwards. I wouldn't expect to see cdi 1.8.1 afterwards.

Comment (slintes) [reply go.sum]: >  I wouldn't expect to see cdi 1.8.1 afterwards.

at least if this was based of master after https://github.com/kubevirt/kubevirt/pull/2767 was merged :)

**Comment by dhiller** [comment]
Closing in favor of #2796

/close

**Pull request closed without merging**